packages/@ourworldindata/grapher/src/controls/SettingsMenu.tsx

```diff
@@ -272,6 +268,10 @@ export class SettingsMenu extends React.Component<{
         return type.replace(/([A-Z])/g, " $1")
     }
 
+    @computed get selectionArray(): SelectionArray {
+        return makeSelectionArray(this.manager)
+    }
```

Comment:
This introduces a bug AFAICS - the mobx dependency is only on this.manager and so when the shared selection is changed outside of this selectionArray will not be recomputed